config/styles/blade-effects.md

```diff
@@ -15,9 +16,17 @@ Note that some of these EFFECTS may be included in features used by some prop fi
 
 The majority of these are self explanitory. 
 Just read EFFECT as "do something during_XXXX"
-### EFFECT_NONE
-Never generated, used as defaults in some places to mean "no effect", but is otherwise not useful for anything.
 
+## Index of Effects
```

Comment:
I think automated left/right-navs would be better.
(It's a little more complicated to do of course, but then we'd have it for all pages.)